lib/main_common.pm

```diff
@@ -1709,8 +1709,11 @@ sub load_extra_tests_docker {
     loadtest "containers/docker";
     loadtest "containers/docker_runc";
     loadtest "containers/containers_3rd_party";
-    loadtest "containers/docker_image" if (!is_public_cloud && (is_sle(">=12-sp3") || is_opensuse));
-    loadtest "containers/docker_compose" unless is_sle('<15');
+    if ((!is_public_cloud() && is_sle(">=12-sp3")) || is_opensuse()) {
+        loadtest "containers/docker_image";
+        loadtest "containers/container_diff";
+    }
+    loadtest "containers/docker_compose" unless (is_sle('<15') || is_sle('=15-sp2'));
```

Comment:
why have you changed this condition? It was just `unless is_sle('<15')`

Reply:
Because `docker-compose` is not present in SLE 15-SP2, nor in it's PackageHub.

Reply:
It's unlikely to come back with SP3, so:

```suggestion
    loadtest "containers/docker_compose" unless (is_sle('<15') || is_sle('>=15-sp2'));
```